### PR TITLE
ruby 3.0.5, 3.1.3, 3.2.1 for Ubuntu 22.10/x86_64 and 22.04/arm64

### PR DESCRIPTION
**Pull request by mathieujobin**
Adds ruby 3.0.5, 3.1.3, 3.2.1 for Ubuntu 22.10/x86_64 (Already uploaded)

image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@
   * 3.2.0-rc1 [\#5280](https://github.com/rvm/rvm/pull/5280)
   * 3.2.0 [\#5283](https://github.com/rvm/rvm/pull/5283)
   * 3.2.1 [\#5306](https://github.com/rvm/rvm/pull/5306)
+  * 3.0.5, 3.1.3 and 3.2.1 For Ubuntu 22.10 / x86_64 [\#5308](https://github.com/rvm/rvm/pull/5308)
```
Comment (raelgc) [inline CHANGELOG.md:14]: The binary section is a bit below, called "Binaries". Please move to that section.

File: config/md5 [hunk 2]
```diff
@@ -1202,6 +1202,18 @@ https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/20.04/x86_64/ruby-3.0.0.tar
...
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.0.4.tar.bz2=65ab3105b0c6d5e8a2e9977a3ab7ed94
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.0.5.tar.bz2=08547ad8e760e0776c74401ae5bbd8e5
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.1.2.tar.bz2=631b4ea065d18df96c6ccbc141904bb3
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.1.3.tar.bz2=03a67e234d886f3c698c1571e42cc7a6
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.2.0.tar.bz2=0ded0cb8ce18a846d71d7e04d24c2e78
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.04/aarch64/ruby-3.2.1.tar.bz2=5251aed0ba4d8349413d6bf4c261bea3
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.10/x86_64/ruby-3.0.5.tar.bz2=f09694c4f23d7625f72606bce0efb710
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.10/x86_64/ruby-3.1.3.tar.bz2=340db3acb58829b51e4a7ac1c77246d4
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.10/x86_64/ruby-3.2.0.tar.bz2=a1bd86cb46a9e6c40c285258879a3d59
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/22.10/x86_64/ruby-3.2.1.tar.bz2=d86b2a1e9721b9459e1283006e03fa92
```
Comment (raelgc) [inline config/md5:1216]: `3.1.3`, `3.2.0` and `3.2.1` are duplicate (see lines `1202`, `1203` and `1204.

Comment (mathieujobin) [reply config/md5:1231]: No, these are for Ubuntu 22.10, Kinetic

File: CHANGELOG.md (hunk 1 above)
Comment (mathieujobin) [reply CHANGELOG.md]: done

**Pull request merged**